.github/workflows/cross-compiles.yml

```diff
@@ -210,3 +210,9 @@ jobs:
         make test HARNESS_JOBS=${HARNESS_JOBS:-4} \
                   TESTS="${{ matrix.platform.tests }} -test_afalg" \
                   QEMU_LD_PREFIX=/usr/${{ matrix.platform.arch }}
+    - name: make evp tests
+      if: github.event_name == 'pull_request' && matrix.platform.tests != 'none'
```

Comment:
Should this also be ( || 'push") && (matrix....)

Reply:
No, in that case the evp tests are run as part of the more complete set.